src/ui/CustomStyleEditor.tsx

```diff
@@ -540,23 +545,21 @@ export class CustomStyleEditor extends React.PureComponent<any, any> {
   }
 
   handleList(val) {
-    //edit mode
-    if (
-      this.state.mode > 0 &&
-      this.isCustomStyleAlreadyApplied() &&
-      false === val.target.checked
-    ) {
+    const selectedStyle = val.target.value;
+    const isList = selectedStyle === 'listStyle';
+    const styleLevel = selectedStyle === 'none' ? 0 : isList ? 1 : 1;
+    if (this.state.mode > 0 && this.isCustomStyleAlreadyApplied() && !isList) {
       this.showAlert();
     } else {
       this.setState((prevState) => ({
+        selectedStyle,
```

Comment:
shouldn't this start with `...prevState` to make sure any other state values not overwritten are preserved?